schemas/aws/account-1.yml

```diff
@@ -2,90 +2,166 @@
 "$schema": /metaschema-1.json
 version: '1.0'
 type: object
+title: "AWS Account"
+description: |
+  Schema for defining an AWS account. This schema captures all metadata,
+  configuration, and ownership information required to manage AWS accounts
+  in app-interface. It supports integration with automation, billing,
+  security, and resource management systems, and enables self-service
+  account management, tagging, and compliance tracking.
 
 additionalProperties: false
 properties:
   "$schema":
     type: string
     enum:
     - /aws/account-1.yml
+
   labels:
     "$ref": "/common-1.json#/definitions/labels"
+
   name:
     type: string
+    description: |
+      The unique name of the AWS account. Used to identify the account
+      across the system.
+
   description:
     type: string
+    description: |
+      A detailed description of the AWS account, including its intended
+      use, environment, or any relevant context.
+
   consoleUrl:
     type: string
     format: uri
+    description: |
+      The URL to access the AWS console for this account.
+
   uid:
     type: string
+    description: |
+      The unique identifier (UID) for the AWS account.
+
   resourcesDefaultRegion:
     type: string
+    description: |
+      The default AWS region for resources created in this account.
+
   supportedDeploymentRegions:
     type: array
     items:
       type: string
-      description: "The regions where the account is enabled and can be used."
+      description: |
+        The regions where the account is enabled and can be used for
```

Comment:
```suggestion
        The region where the account is enabled and can be used for
```

Reply:
This is intentionally plural. The property is `supportedDeploymentRegions`, not `supportedDeploymentRegion`. This array can support multiple regions.